package.json

```diff
@@ -27,12 +27,16 @@
   ],
   "main": "dist/index.js",
   "module": "src/index.js",
+  "types": "types/index.d.ts",
   "files": [
     "dist",
+    "types/index.d.ts",
     "src/index.js"
   ],
   "dependencies": {
+    "@types/hast": "^2.0.0",
     "extend": "^3.0.0",
+    "unified": "^9.0.0",
```

Comment:
I believe we usually don’t want to add `unified` as a dependency, because this is typically already used with unified, either directly or because of remark.

```suggestion
```

The rest looks good to me! :+1: